app/src/main/java/de/danoeh/antennapod/activity/MediaplayerActivity.java

```diff
@@ -613,35 +609,28 @@ public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
         }
         if (fromUser) {
             prog = progress / ((float) seekBar.getMax());
+            int duration = controller.getDuration();
             TimeSpeedConverter converter = new TimeSpeedConverter(controller.getCurrentPlaybackSpeedMultiplier());
-            int position = converter.convert((int) (prog * controller.getDuration()));
-            txtvSeek.setText(Converter.getDurationStringLong(position));
+            int position = converter.convert((int) (prog * duration));
+            txtvPosition.setText(Converter.getDurationStringLong(position));
+
+            if (showTimeLeft) {
+                int timeLeft = converter.convert(duration - (int) (prog * duration));
+                txtvLength.setText("-" + Converter.getDurationStringLong(timeLeft));
+            }
         }
     }
 
     @Override
     public void onStartTrackingTouch(SeekBar seekBar) {
-        cardViewSeek.setScaleX(.8f);
-        cardViewSeek.setScaleY(.8f);
-        cardViewSeek.animate()
-                .setInterpolator(new FastOutSlowInInterpolator())
-                .alpha(1f).scaleX(1f).scaleY(1f)
-                .setDuration(200)
-                .start();
+
```

Comment:
I think something went wrong here while merging

Reply:
Check this also.